sdk/test/UnitTests/EfCoreTests.cs

```diff
@@ -86,6 +86,7 @@ public void Test_EFCore_unsuccessful_query()
         }
 
         [TestMethod]
+        [Obsolete]
```

Comment:
Why is this test marked as obsolete?

Reply:
context.Database.ExecuteSqlCommand is obsolete.  I've updated to context.Database.ExecuteSqlInterpolated, which is one of the recommended resolutions, and removed the obsolete marker.